File: app/src/main/java/com/droidtitan/volleyexamples/rest/model/air/AirQualityHolder.java
package com.droidtitan.volleyexamples.rest.model.air;

import com.google.gson.annotations.SerializedName;

public class AirQualityHolder {

    @SerializedName("reporte")
    int hourOfReport;

    @SerializedName("calidad")
    AirQuality airQuality;

    @SerializedName("clima")
    Weather weather;

    @SerializedName("uv")
    Uv uv;

    @SerializedName("zonas")
    Zones zones;

    public int getHourOfReport() {
        return hourOfReport;
    }

    public AirQuality getAirQuality() {
        return airQuality;
    }

    public Weather getWeather() {
        return weather;
    }

    public Uv getUv() {
        return uv;
    }

    public Zones getZones() {
        return zones;
    }
}

File: app/src/main/java/com/droidtitan/volleyexamples/rest/model/air/AirQualityHolder.kt
package com.droidtitan.volleyexamples.rest.model.air

import com.google.gson.annotations.SerializedName

public class AirQualityHolder {

    SerializedName("reporte")
    var hourOfReport: Int = 0

    SerializedName("calidad")
    var airQuality: AirQuality? = null

    SerializedName("clima")
    var weather: Weather? = null

    SerializedName("uv")
    var uv: Uv? = null

    SerializedName("zonas")
    var zones: Zones? = null
}
